sources/editor/Xenko.Assets.Presentation/AssetEditors/Gizmos/GizmoViewportRenderer.cs

```diff
@@ -17,7 +17,7 @@ class GizmoViewportRenderer : SceneRendererBase
         /// <summary>
         /// The inner compositor to draw inside the viewport.
         /// </summary>
-        public ISceneRenderer Content { get; set; }
+        new public ISceneRenderer Content { get; set; }
```

Comment:
Yes, let's rename that one to something else. `SceneRenderer` sounds good.

Reply:
A rename here is fine since it doesn't impact asset serialization (only happens on editor side). So minor version is fine too.

But I agree that we can do it after that PR since it is a bit out of scope of just fixing warnings.